iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/condition_variable_waiter.hpp

```diff
@@ -42,6 +42,11 @@ class ConditionVariableWaiter
     /// @brief Waits until notify is called on the ConditionVariableSignaler
     void wait() noexcept;
 
+    /// @brief Was the ConditionVariableWaiter triggered by a ConditionVariableSignaler. Does not change the state in
+    /// contrast to wait or timedWait!
+    /// @return true if it was triggered otherwise false
+    bool wasTriggered() const noexcept;
```

Comment:
We use `notifyOne()` with a signaler. The term trigger is used for the conditions. Shall we have another name here as it is a bit confusing. `wasNotified()`?